spec/api-utility-process-spec.ts

```diff
@@ -215,7 +215,8 @@ describe('utilityProcess module', () => {
       });
       await once(child, 'spawn');
       expect(child.kill()).to.be.true();
-      await once(child, 'exit');
+      const [code] = await once(child, 'exit');
+      expect(code).to.equal(0);
```

Comment:
Can we get a test for non-zero exit codes via `process.exit(42)` or similar?

Reply:
It should be covered in https://github.com/electron/electron/blob/dd50afa8c2d82f4a448bd66675fcde2d64642031/spec/api-utility-process-spec.ts#L123-L128

Reply:
Thanks, I missed that!